.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
-- [ ] closes #xxxx
-- [ ] tests added / passed
```

Comment:
this one shouldn't change, why have you changed the `-` to `*`?

Reply:
That was a mistake from my part sorry